.circleci/config.yml

```diff
@@ -60,10 +50,10 @@ jobs:
           name: Build the USWDS package
           command: npm run release
       - run:
-          name: Publish to NPM
+          name: Publish to NPM with `uswds-2.x` tag
           command: |
             npm config set "//registry.npmjs.org/:_authToken=$NPM_TOKEN"
-            npm publish
+            npm publish --tag uswds-2.x
```

Comment:
Publish with a tag to prevent this v2 release appearing as `latest` in npm. This is parallel to what we did with v1. See https://www.npmjs.com/package/uswds?activeTab=versions